Whisky/Views/BottleCreationView.swift

```diff
@@ -20,11 +22,24 @@ struct BottleCreationView: View {
                 Spacer()
             }
             Divider()
-            HStack {
+            HStack(alignment: .top) {
                 Text("create.name")
                 Spacer()
-                TextField("", text: $newBottleName)
-                    .frame(width: 180)
+                VStack(alignment: .leading) {
+                    TextField("", text: $newBottleName)
+                        .onChange(of: newBottleName) { _ in
+                            invalidBottleName = false
+                        }
```

Comment:
Validation on *every* character entry gives a really bad user experience for the user.
Imagine they have a Bottle called "Bottle" - with this logic, entering "Bottle" as they try to enter "Bottle1" would immediately throw an error as soon as they typed it.
It's much better to debounce validation, so that it happens around 0.8s after they've entered a character, and if they enter a *new* character, the previous attempt at validation is cancelled and doesn't occur as the new validation takes over.

Reply:
The name validation only happens when you press the Create/Rename button

Reply:
Ahhh, my mistake, I see!